lib/execution_engine2/db/MongoUtil.py

```diff
@@ -211,17 +211,17 @@ def cancel_job(self, job_id=None, terminated_code=None):
         """
         #TODO Should we check for a valid state transition here also?
         #TODO Make cancel code mandatory and part of spec?
+        #TODO Should make terminated_code default to something else, and update clients in Narrative?
         :param job_id: Cancel job by id
         :param terminated_code: Default to terminated by user
-        :param job: Cancel job with actual job document instance
         """
 
         with self.mongo_engine_connection():
             j = self.get_job(job_id, projection=None)
             self.check_if_already_finished(j.status)
             if terminated_code is None:
                 terminated_code = TerminatedCode.terminated_by_user.value
-
+            j.terminated = time.time()
```

Comment:
model doesn't need to be updated to include this field?

Reply:
That gets fixed https://github.com/kbase/execution_engine2/pull/101/files#r345449369m, ill fix it here too